mesatee_services/kms/proto/src/lib.rs

```diff
@@ -19,34 +19,14 @@
 #[macro_use]
 extern crate sgx_tstd as std;
 
+#[cfg(feature = "mesalock_sgx")]
+use std::prelude::v1::*;
+
+mod kms;
 pub use crate::kms::*;
 pub use crate::proto::KMSClient;
-mod kms;
-pub mod proto {
-    #![allow(warnings)]
-    #![allow(clippy)]
-    #![allow(unknown_lints)]
-    include!("prost_generated/kms_proto.rs");
-}
-
-mod base64_coder {
-    use base64;
-    use serde::{de, Deserialize, Deserializer, Serializer};
-    #[cfg(feature = "mesalock_sgx")]
-    use std::prelude::v1::*;
 
-    pub fn serialize<S>(bytes: &[u8], serializer: S) -> Result<S::Ok, S::Error>
-    where
-        S: Serializer,
-    {
-        serializer.serialize_str(&base64::encode(bytes))
-    }
-
-    pub fn deserialize<'de, D>(deserializer: D) -> Result<Vec<u8>, D::Error>
-    where
-        D: Deserializer<'de>,
-    {
-        let s = <&str>::deserialize(deserializer)?;
-        base64::decode(s).map_err(de::Error::custom)
-    }
+pub mod proto {
+    #![allow(clippy::all)]
+    include!(concat!(env!("OUT_DIR"), "/kms_proto.rs"));
```

Comment:
If I want to serialize and deserialize the ```vec<u8>``` with base64, should I add the serializer and  deserializer here or in ```prost_common```

Reply:
It's better to put in the generator.

BTW, I suggest to use hex to encode/decode u8. Firstly, no need to include third-party crates (`base64`) and we already have a implementation in `teaclave_config`. Secondly, it reads better than base64.

Reply:
Thx. I agree. hex will be better.